src/components/AvailabilityScheduleSelector/AvailabilityScheduleSelector.tsx

```diff
@@ -29,15 +29,36 @@ function AvailabilityScheduleSelector(
     } = props;
 
     const handleRenderDateCell = (
-        time: Date, // being used internally, somehow -Jorge (laughed after I literally put what he said)
+        time: Date,
         selected: boolean,
         refSetter: (dateCell: HTMLElement | null) => void
     ) => {
+        const currentTime = time.getHours() + time.getMinutes() / 60;
         const selectionId = selected ? 'selectedCell' : 'unSelectedCell';
 
-        return <div id={selectionId} ref={refSetter} />;
+        if (startTime === 0) {
+            if (currentTime < endTime) {
+                return <div id={selectionId} ref={refSetter} />;
+            }
+
+            return <div />;
+        }
+
+        if (currentTime < endTime && Math.floor(currentTime) !== 0) {
+            return <div id={selectionId} ref={refSetter} />;
+        }
+        return <div />;
     };
 
+    //     time: Date, // being used internally, somehow -Jorge (laughed after I literally put what he said)
```

Comment:
Is there a reason why these lines are commented out?

Reply:
I was there when Sophia was working on this. She deleted the commented-out lines, I think she just needs to push those changes perhaps.

Reply:
Okay -- typically, unless there is a good reason for leaving old code commented, it should be deleted

Reply:
yeah, it was there before I started working on it. I'll delete it and commit again